src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -426,16 +426,38 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environment and system noise. Their
-collection is RECOMMENDED, before/during/after each session. This data is stored
-inside a subject folder named `sub-emptyroom`. The `session label` SHOULD be
-that of the date of the empty-room recording (e.g. `ses-YYYYMMDD`). The
-`scans.tsv` file containing the date/time of the acquisition SHOULD also be
-included. Hence, users will be able to retrieve the empty-room recording that
-best matches a particular session with a participant, based on date/time of
-recording.
+Empty-room MEG recordings capture the environmental and recording system's
+noise.
```

Comment:
It's not clear to me if the rest of the sidecars should also be stored for empty room? For example, channels.tsv?

Reply:
yes, JSON and TSV files should be present. It should be possible to query noise recordings just like subject recordings. Do you think that this should be explicitly added to the text? 

Do you know if there is any reference in the specification to phantom (i.e. bottle) recordings with the MRI? If so, we could try to make it more similar between MEG and MRI.

Reply:
I added this to the examples and clarified in-text: https://github.com/bids-standard/bids-specification/pull/480/commits/e4c04f8789684c84c405d9d04f01d2648fa6cd35

Reply:
Good point about the phantom recordings in MRI. I couldn't find anything in the spec related to that. Maybe we can tag one of the MRI folks?

Reply:
@bids-standard/raw-mri does anyone of you know something about specifying MRI phantom recordings in BIDS? I.e., recordings to capture the noise of the system?

Reply:
There's nothing currently addressing it, but intuitively, a phantom seems obviously like a subject, and I would record all of the pertinent metadata to enable contrasting.

What's the specific question here? Whether to downgrade REQUIRED metadata in these cases?